src/main/java/nisum/marketplace/backend/CartItemsController.java

```diff
@@ -0,0 +1,119 @@
+package nisum.marketplace.backend;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import nisum.marketplace.backend.model.CartItems;
+import nisum.marketplace.backend.service.CartItemsService;
+
+@CrossOrigin
+/*localhost:8080/api/marketplace/(the mapping I defined)*/
+@RequestMapping("/api/marketplace")
+@RestController
+public class CartItemsController {
+	
+	@Autowired
+	CartItemsService cartService;
+	
+	/*
+	 * No requestbody and path variable needed
+	 * */
+	@GetMapping("/getcart")
+	public ResponseEntity<Object> getCart(){
+		try {
+			//var  has type CartItems
+			var cartList = cartService.getCart();
```

Comment:
I think you should declare the type instead of using var for type safety

Reply:
i fixed this one